backend/src/main/java/com/engineering/thesis/backend/repository/AppointmentRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.engineering.thesis.backend.repository;
+
+import com.engineering.thesis.backend.model.Appointment;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AppointmentRepository extends JpaRepository<Appointment,Long> {
+}
```

Comment:
Co nas powstrzymuje przed zrobieniem 
```
@Repository
public interface AppointmentRepository extends JpaRepository<Appointment,Long> {
}

@Repository
public interface DoctorRepository extends JpaRepository<Doctor,Long> {
}

etc...
``` ?

I think that if there's no underlying logic then we can go for repositories.java file and just put them there :)

Reply:
Powstrzymuje nas to co wcześniej wspominałam o Domian Driven Design. Jakbyśmy wszystko wrzucali w jedno to w końcu mogłoby nam się pomieszać.

Reply:
Pozostawione bez zmian